doc/devel/gitwash/development_workflow.rst

```diff
@@ -158,6 +158,142 @@ When you are ready to ask for the merge of your code:
+repository is called `matplotlib`.
+
+The first step is to make sure that your local copy of the upstream repository is
+up-to-date::
+
+     $ git fetch matplotlib
+
+This updates your local copy of the repository, but does not change any files
+in your working copy.  Next, switch to the branch that you want to update::
+
+     $ git checkout backend_plt_refactor
+
+You are now ready to start the rebase of your branch onto the target
+parent branch, in this case `matplotlib/master` ::
+
+     $ git rebase matplotlib/master
+
+and git will then give a bunch of feed back::
+
+     First, rewinding head to replay your work on top of it...
+     Applying: first steps to extract FigureManager* and friends from pyplot
+     Applying: split backend_qt4 into two parts, with and without Gcf
+     ...
+     Applying: pep8 clean up on backend_gtk3.py
+     Using index info to reconstruct a base tree...
+     M       lib/matplotlib/backends/backend_gtk3.py
+     Falling back to patching base and 3-way merge...
+     Auto-merging lib/matplotlib/backends/backend_gtk3.py
+     CONFLICT (content): Merge conflict in lib/matplotlib/backends/backend_gtk3.py
+     Failed to merge in the changes.
+     Patch failed at 0013 pep8 clean up on backend_gtk3.py
+     The copy of the patch that failed is found in:
+        /home/tcaswell/other_source/matplotlib/.git/rebase-apply/patch
+
+     When you have resolved this problem, run "git rebase --continue".
+     If you prefer to skip this patch, run "git rebase --skip" instead.
+     To check out the original branch and stop rebasing, run "git rebase --abort".
+
+A number of commits could be cleanly applied to
+the tip of `matplotlib/master`,  however, git eventually hits a commit
+that had conflicts.  In this case in the file
+`lib/matplotlib/backends/backend_gtk3.py`.  For more verbose information run ::
+
+     $ git status
+
+     You are currently rebasing branch 'backend_plt_refactor' on 'e6f8993'.
+       (fix conflicts and then run "git rebase --continue")
+       (use "git rebase --skip" to skip this patch)
+       (use "git rebase --abort" to check out the original branch)
+
+     Unmerged paths:
+       (use "git reset HEAD <file>..." to unstage)
+       (use "git add <file>..." to mark resolution)
+
+             both modified:      lib/matplotlib/backends/backend_gtk3.py
+
+     no changes added to commit (use "git add" and/or "git commit -a")
+
+This tells you exactly where the conflict is and provides some advice
+on how to proceed.  Opening up the file in question, you will see
+blocks that look something like this::
+
+     <<<<<<< HEAD
+     =======
+             self.__dict__.clear()   # Is this needed? Other backends don't have it.
+     >>>>>>> pep8 clean up on backend_gtk3.py
+
+The block of code between `<<<<<<<` and `=======` is the code on the
+target branch (in this case nothing) and the code between `=======`
+and `>>>>>>>` is the code on your branch.  The rest of the code is the
```

Comment:
It's not "on your branch", but "on the last commit that was attempted to be rebased". This might be a bit difficult to convey, but it could cause confusion if there's a conflict in the middle on lines that were subsequently changed.